eclipse/plugins/org.switchyard.tools.ui.bpmn2/META-INF/MANIFEST.MF

```diff
@@ -21,9 +21,9 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.emf.edit.ui,
  org.eclipse.emf.transaction,
  org.eclipse.bpmn2,
- org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.ui;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.runtime.jboss.jbpm;bundle-version="[1.2.3,1.4.0)",
+ org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.5.0)",
```

Comment:
yes 1.4.0) means that we don't want to use higher version than 1.4.0 included.
my question was not about that. it was why we put an upper restriction for the bpmn2 modeler plugins?